docs/docs/api_documentation_style.md

```diff
@@ -105,68 +125,73 @@ In order, the sections of a funciton comment are:
 	
 	Function handles have type `function_handle`.
 
-5. Returns
+2. Returns
 
-   Explanation of the returned values and their types. We distinguish two cases.
+    Explanation of the returned values and their types. We distinguish two cases.
    
-   First, when a single value is returned, we use the Google style:
+    First, when a single value is returned, we use the Google style:
    
-   ```matlab
-   function c = sum(a, b)
-   % Sums two numbers
-   %
-   % Returns:
-   %   double: The sum of the parameters
-   ```
+    ```matlab
+    function c = sum(a, b)
+    % Sums two numbers
+    %
+    % Returns:
+    %   double: The sum of the parameters
+    ```
    
    
-   ```matlab
-   function [c d] = sorted2(a, b)
-   % Sorts two numbers
-   %
-   % Returns
-   % -------
-   %   c: double
-   %     Smallest number
-   %   d: double
-   %     Largest number
-   ```
+    ```matlab
+    function [c d] = sorted2(a, b)
+    % Sorts two numbers
+    %
+    % Returns
+    % -------
+    %   c: double
+    %     Smallest number
+    %   d: double
+    %     Largest number
+    ```
    
-6. Raises
+3. Raises (optional)
 
 	An optional section detailling which errors get raised and under what conditions.
 	
 	TO BE COMPLETED
 	
-7. Warns
+4. Warns (optional)
 
 	An optional section detailling which warnings get raised and under what conditions, formatted similarly to Raises.
 	
-8. Warnings
+5. Warnings (optional)
 
 	An optional section with cautions to the user in free text/reST.
+
+6. Examples (encouraged)
+
+	A section with explicit commands illustrating as clearly as possible one or several ways of calling the function.
+
+	(TODO: allow examples that would be executed by the test framework, possibly with doctests, to be tested)
 	
-9. See Also
+7. See Also (encouraged)
```

Comment:
The See Also has a particular weird format in Matlab (function names in all caps it seems?). I suggest following the Sphinx conventions if we can't get both to agree in some way.